kaspresso/src/main/kotlin/com/kaspersky/kaspresso/device/video/VideosImpl.kt

```diff
@@ -12,7 +12,10 @@ class VideosImpl(
 ) : Videos {
 
     override fun record(tag: String) {
-        val videoFile: File = resourceFilesProvider.provideVideoFile(tag)
+        val sanitizedVideoName = tag
+            .replace("[/:\"*?<>|]+".toRegex(), "")
```

Comment:
1. The issue creator used backing slash to cover each special simbol. It's not needed?
2. Please, replace with the undescrore. User expected a simbol there
3. Please, if the video name changed, then leave a warning log with something like `log("Can't record video with name $tag since it contains one of the following special characters [/:\"*?<>|]. Changing the name to $sanitizedVideoName")`